static/src/javascripts/bootstraps/enhanced/main.js

```diff
@@ -179,7 +179,9 @@ const bootEnhanced = (): void => {
     }
 
     if (
-        (config.isMedia || qwery('video, audio').length) &&
+        (config.isMedia ||
+            qwery('video, audio').length ||
+            $('.youtube-media-atom').length > 0) &&
```

Comment:
Oh wow. This is an expensive switch! Why not combining both of them into one query `qwery('video, audio, .youtube-media-atom').length`? We should also wrap this into fastdom, if it's available at this point.

What do you think?